projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -150,7 +150,7 @@ export default function MyJetpackScreen() {
 							<GlobalNotice message={ message } options={ options } clean={ clean } />
 						</Col>
 					) }
-					{ isStatsModuleActive && (
+					{ showJetpackStatsCard && (
```

Comment:
In theory, we'd like to show the stats card all the time, even if the main Jetpack plugin is not installed (like we do with all the other cards)

However, after tinkering with this a bit, I am realizing that there a some other implications that we'd need to work out in doing this. For example, what to show on the card CTA when there is no site connection? "Learn more"? "Fix site connection"?

I think we can leave this as-is in this PR and work on having the stats card show up for other standalone plugins in another PR